nussknacker-dist/src/universal/conf/base-application.conf

```diff
@@ -2,7 +2,7 @@ storageDir: ./storage
 storageDir: ${?STORAGE_DIR}
 
 jdbcServer {
-  dbFilePath: ${storageDir}"/db"
+  dbFilePath: ${storageDir}"/db/nk"
```

Comment:
nope, don't do it - it is path used in dockers etc. you will blow app somewhones storage...

Reply:
I'v left  dbFilePath: ${storageDir}"/db/"

Reply:
ok, but also you haven't used it anywhere?

Reply:
Because now I can change dbFiilePath by set DB_FILE_PATH in env:
```
jdbcServer {
  dbFilePath: ${dbFilePath}
  dbName: ${dbName}
  user: ${dbUser}
  password: ${dbPassword}
}
```

Reply:
I know that you can, but haven't seen that you set it somewhere. You've changed it in your IDE only?

Reply:
Exactly.